test/retryable_unit_test.py

```diff
@@ -0,0 +1,34 @@
+import common
+import os
+import unittest
+
+# This class patches in to the Python unittest TestCase object to incorporate
+# support for an environment variable EMTEST_RETRY_COUNT=x, which enables a
+# failed test to be automatically re-run to test if the failure might have been
+# due to an instability.
+class RetryableTestCase(unittest.TestCase):
+  def run(self, result=None):
+    self.origTestMethodName = self._testMethodName
```

Comment:
Is this used anywhere?

Reply:
Thanks, that was leftover from testing. Removed.